documentation/guide-configuration.asciidoc

```diff
@@ -156,47 +156,189 @@ We recommend the following base layout for the hierarchical business configurati
 Often you need to have passwords (for databases, third-party services, etc.) as part of your configuration. These are typically environment specific (see above). However, with DevOps and continuous-deployment you might be tempted to commit such configurations into your version-control (e.g. `git`). Doing that with plain text passwords is a severe problem especially for production systems. Never do that! Instead we offer some suggestions how to deal with sensible configurations:
 
 === Password Encryption
-A simple but reasonable approach is to configure the passwords encrypted with a master-password. The master-password should be a strong secret that is specific for each environment. It must never be committed to version-control.
-In order to support encrypted passwords in spring-boot `application.properties` all you need to do is to add https://github.com/ulisesbocchio/jasypt-spring-boot#jasypt-spring-boot[jasypt-spring-boot] as dependency in your `pom.xml`(please check for recent version):
-[source, xml]
+Let us have a look how to encrypt password using `jasypt-springboot-starter` and `jasypt-springboot`.
+
+You can also have a look at their page link:http://www.jasypt.org/index.html[Jasypt].
+
+==== How to Use Jasypt-SpringBoot-Starter
+The first step would be to add the starter dependency to our `pom.xml` file.
+
+[source,xml]
 ----
 <dependency>
-  <groupId>com.github.ulisesbocchio</groupId>
-  <artifactId>jasypt-spring-boot-starter</artifactId>
-  <version>1.17</version>
+    <groupId>com.github.ulisesbocchio</groupId>
+    <artifactId>jasypt-spring-boot-starter</artifactId>
+    <version>2.0.0</version>
 </dependency>
 ----
+
+Please check the latest version link:https://mvnrepository.com/artifact/com.github.ulisesbocchio/jasypt-spring-boot-starter[here] .
+
 This will smoothly integrate http://jasypt.org/[jasypt] into your https://projects.spring.io/spring-boot/[spring-boot] application. Read this https://wiki.jasig.org/display/CASUM/HOWTO+Use+Jasypt+to+encrypt+passwords+in+configuration+files[HOWTO] to learn how to encrypt and decrypt passwords using jasypt. Here is a simple example output of an encrypted password (of course you have to use strong passwords instead of `secret` and `postgres` - this is only an example):
+
 [source, bash]
 ----
-----ARGUMENTS-------------------
+ARGUMENTS
```

Comment:
just for clarification:
This was the raw output of jasypt CLI.
Has this been changed, as I produced invalid ASCIIDOC since the 5 dashes indicate the end of the source?
We could also simply change the 5 dashes opening and ending to 3 backticks instead.

Reply:
should we also replace the example data to fit with the new encryption algorithm?

Reply:
@hohwille  do you mean example provided here in the document or any other application already there for jasypt?

Reply:
I was only talking about the outlook of jasypt CLI where I annotated this review comment. With example I am talking about the used data
```
input: postgres
password: secret
```
and especially the produced result that IMHO will change as the suggested security algorithm changed with this PR. Also a question would be if the CLI changes or a different Main-Class needs to be used for this now.